pkg/secrets/secrets.go

```diff
@@ -82,6 +83,7 @@ func generateUserDataWithPubKey(pubKey string) string {
 			$firewallRuleName = "ContainerLogsPort"
 			$containerLogsPort = "10250"
 			New-NetFirewallRule -DisplayName $firewallRuleName -Direction Inbound -Action Allow -Protocol TCP -LocalPort $containerLogsPort -EdgeTraversalPolicy Allow
+			New-NetFirewallRule -DisplayName "WindowsExporter" -Direction Inbound -Action Allow -Protocol TCP -LocalPort "` + windowsExporterPort + `" -EdgeTraversalPolicy Allow
```

Comment:
Was the previous of defining vars inside the return not ideal? If that's the case, I don't expect them to be fixed in this PR. Just want to know the reasoning behind different patterns.

Reply:
I'd prefer to have them outside so we can make them reusable variables. Did not want to refactor that in this PR.